app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/ui/tracker_activity/DeviceShieldTrackerActivityViewModel.kt

```diff
@@ -157,6 +158,18 @@ class DeviceShieldTrackerActivityViewModel @Inject constructor(
         }
     }
 
+    override fun setDismissed() {
+        vpnStore.dismissNotifyMeInAppTp()
+    }
+
+    override fun isDismissed(): Boolean {
+        return vpnStore.isNotifyMeInAppTpDismissed()
+    }
+
+    override fun visibilityChanged(visible: Boolean) {
+        // noop
```

Comment:
No implementation needed here as there is nothing to do in case the visibility changed. This makes sense e.g when the view is added to a recyclerview.